napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -355,27 +409,46 @@ def __init__(
         enabled: bool = True,
         installed: bool = False,
         npe_version=1,
+        versions_conda: list = [],
+        versions_pypi: list = [],
     ):
         super().__init__(parent)
+        self.url = url
+        self._versions = {}
+        self._versions['Conda'] = versions_conda
+        self._versions['PyPI'] = versions_pypi
         self.setup_ui(enabled)
         self.plugin_name.setText(package_name)
+
+        self._populate_version_dropdown('PyPI')
         self.package_name.setText(version)
-        self.summary.setText(summary)
-        self.package_author.setText(author)
+        if summary:
```

Comment:
Just curious about this change, does it save space to not set the text at all when we have no summary or author?

Reply:
The package author and summary were changed to eliding labels.  They don't like none types and so I would get this error:
```
TypeError: 'NoneType' object is not subscriptable
```